tests/integration/test_cases/inference/chat_completion.json

```diff
@@ -50,7 +50,7 @@
           "parameters": {
             "location": {
               "param_type": "string",
-              "description": "The city and state, e.g. San Francisco, CA"
+              "description": "The city and state, e.g. San Francisco, CA. Both city and state are required."
```

Comment:
nit: Can you try "The city and state (both required), e.g. San Francisco, CA."?

Reply:
Ahh, that's much cleaner! I updated this to use this briefer syntax, ran the tests multiple times locally, and they passed every time without flaking. Thanks for the prompt engineering help :smile: